packages/core/lib/commands/debug.js

```diff
@@ -30,16 +30,16 @@ const command = {
   },
   help: {
     usage:
-      "truffle debug [<transaction_hash>] [--fetch-external]" + OS.EOL +
-      "                             [--compile-tests|--compile-all|--compile-none]",
+      "truffle debug [<transaction_hash>] [(--fetch-external|-x)]" + OS.EOL +
+      "                             [(--compile-tests|--compile-all|--compile-none)]",
```

Comment:
This seems a bit congested. The `--fetch-external`, `|` and the `-x` are squashed together. The `l|` is particularly distracting. Adding a space looks better? I think @gnidan has stronger typography opinions than me :)

```
  Usage:        truffle debug [<transaction_hash>] [(--fetch-external | -x)]
                             [(--compile-tests | --compile-all | --compile-none)] [--network <name>] [--config <file>]

```

Reply:
I am ok with doing either one. Let's just make sure everyone is happy with whatever standard we pick. Here are two screenshots for comparison.
<img width="1433" alt="Screen Shot 2021-07-22 at 5 01 34 PM" src="https://user-images.githubusercontent.com/14827965/126709190-0e5a76e2-9ebc-4f5d-a350-3dbb8cc18f1f.png">
<img width="1440" alt="Screen Shot 2021-07-22 at 5 02 36 PM" src="https://user-images.githubusercontent.com/14827965/126709196-5a988b03-106f-4360-a8c9-c15fde39e641.png">

Reply:
Do we really need parentheses inside brackets?  The brackets already act as a grouper; there's not a need for a redundant second grouper inside that.

Reply:
The consensus is no space around the pipes? I still think `--fetch-external|-x` is a bit wonky.

Reply:
@cds-amal We are using no parens and no space around the pipe.